packages/app/src/components/tenderbox/index.tsx

```diff
@@ -36,7 +36,10 @@ const TenderBox: FC<BoxExtendedProps> = (props) => {
           width: "220px",
         }}
       />
+      <Box height={props.height} width={props.width} fill style={{overflow: "hidden !important", borderRadius: "50px"}}>
```

Comment:
My guess is there has to be a more idiomatic way of achieving this, I can have a look at it later